guides/plugins/plugins/bundle.md

```diff
@@ -0,0 +1,109 @@
+---
+nav:
+  title: Bundle
+  position: 20
+
+---
+
+# Bundle
+
+Plugins are based on the Symfony bundle concept, but offer additional features like lifecycle events and the ability to be managed in the Shopware administration. This is maybe unwanted in some cases, like project critical customizations which should not be managed via the Shopware administration. In this case, you can use a Symfony bundle instead of a plugin.
+
+The Bundle can be put into the `src/` folder of your project and needs to be registered in the `config/bundles.php` file of your project.
+
+## Choosing the right Bundle class
+
+There are two Bundle classes you can choose from:
+
+- `Shopware\Core\Framework\Bundle`
+- `Symfony\Component\HttpKernel\Bundle\Bundle`
+
+The first one is the Shopware bundle class and the second one is the Symfony bundle class. The Shopware bundle class extends the Symfony bundle class, but offers additional features like to be an Theme, bring JavaScript/CSS files, Migrations, etc. If you don't need these features, you can use the Symfony bundle class instead.
+
+## Creating a Bundle
+
+In the Shopware project template it is intended to put the bundle into the `src/` folder of your project and use the namespace `App\`.
+
+```php
+// <project root>/src/YourBundleName.php
+<?php declare(strict_types=1);
+
+namespace App\YourBundleName;
+
+use Shopware\Core\Framework\Bundle;
+
+class YourBundleName extends Bundle
+{
+}
+```
+
+The bundle class needs to be registered in the `config/bundles.php` file of your project.
+
+```php
+// <project root>/config/bundles.php
+//...
+App\YourBundleName\YourBundleName::class => ['all' => true],
+//...
+```
+
+## Adding services, twig templates, routes, theme, etc
+
+You can add services, twig templates, routes, etc. to your bundle like you would do in a plugin. Just create `Resources/config/services.xml` and `Resources/config/routes.xml` files or `Resources/views` for twig templates. The bundle will be automatically detected and the files will be loaded.
+
+To mark your bundle as a theme, it's enough to implement the `Shopware\Core\Framework\ThemeInterface` interface. This will automatically register your bundle as a theme and make it available in the Shopware administration. You can also add a `theme.json` file to define the theme configuration like [described here](../themes/theme-configuration.md).
```

Comment:
```suggestion
To mark your bundle as a theme, it's enough to implement the `Shopware\Core\Framework\ThemeInterface` interface in your bundle class.
This will automatically register your bundle as a theme and make it available in the Shopware administration.
You can also add a `theme.json` file to define the theme configuration like [described here](../themes/theme-configuration.md).
```